lib/kwtool_web/templates/keyword/show.html.eex

```diff
@@ -0,0 +1,85 @@
+<h1>Result for "<%= @keyword.phrase %>"  <span class="badge status--added"><%= @keyword.status %></span></h1>
+
+<div class="keyword-details">
```

Comment:
Let's keep the class name singular as per our convention 👍 

```suggestion
<div class="keyword-detail">
```

Reply:
Updated! Thanks 👍